config/jobs/kubernetes-sigs/lws/lws-presubmit.yaml

```diff
@@ -0,0 +1,32 @@
+presubmits:
+  kubernetes-sigs/lws:
+  - name: pull-lws-test-main
```

Comment:
this will only run the unit tests, we need another list item for the integration tests "make test-integration" like the jobset one: https://github.com/kubernetes/test-infra/blob/4a4a79ceb2fecfae58be06264f74631619951d21/config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml#L35

Reply:
the lws repo will run both unit/integration test for command "make test", do we need to separately run the integration/unit test?

Reply:
Splitting them make it easier, but I guess we can do that as a followup.

Reply:
sg, thanks!